.github/workflows/tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/vendor/bundle
-          key: ${{ runner.os }}-${{ matrix.ruby }}-unit-gems-${{ hashFiles('apps/*/Gemfile.lock', 'Gemfile.lock') }}-1
+          key: ${{ runner.os }}-${{ matrix.ruby }}-unit-gems-${{ hashFiles('apps/*/Gemfile.lock', 'Gemfile.lock') }}-2
```

Comment:
Not really a part of this, but tests started failing with error, so we need to update those gems.

```
LoadError: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found 
(required by /home/runner/vendor/bundle/ruby/3.0.0/gems/date-3.3.4/lib/date_core.so) 
- /home/runner/vendor/bundle/ruby/3.0.0/gems/date-3.3.4/lib/date_core.so
```